chore(release): 5.0.2-alpha.0 (#9601)
OpenTrons/opentrons-api: app-shell/build/release-notes.md

- More visibility into the progress of protocol execution on the robot
- Quick and easy re-running of your last used protocol

## 5.0.1 to 5.0.2

> The 5.0.2 hotfix release addresses a few bugs. The first fix allows python protocols that load custom labware inline to still take advantage of the new Labware Position check flow. The Second allows users to inspect their custom labware labels within the Labware Setup step in the Opentrons App. Lastly, if run record creation fails on protocol upload to a robot, the error will be surfaced to users as opposed to failing silently.

Reviewer: ```suggestion
The 5.0.2 hotfix release fixes three issues:

1. If you use `load_labware_from_definition` in your Python protocol, you are now able to use Labware Position Check.
2. User-defined labware labels, if present, are now displayed in "Labware Setup"
3. Certain types of protocol upload failures will now show an error message instead of silently failing.
```

## 5.0.0 to 5.0.1

The 5.0.1 hotfix release contains a small fix to the Labware Position Check to appropriately handle protocols that load pipettes, but never pick up tips with that pipette.

## Known Issues

- Your last run protocol will not be saved through robot reboots

> - The expected mapping of protocol load order to usb port info, in the case of multiple instances of a single module type in a protocol, is inconsistent with the messaging in the Opentrons App.

Reviewer: ```suggestion
- When you load multiple instances of the same module type in a protocol, the USB port numbers shown in "Module Setup" are incorrect.
```
